### PR TITLE
Get more 200 favorites

### PR DESCRIPTION
**Pull request by chaspy**
ref: https://github.com/chaspy/favsearch/issues/6

### CODE REVIEW

**Comment by chaspy** [comment]
@kamontia 
please review and merge

File: app.rb [hunk 1]
```diff
@@ -42,7 +42,10 @@
 get '/api/v1.0/favorite' do
   @twitter.access_token = session[:twitter_oauth].token
   @twitter.access_token_secret = session[:twitter_oauth].secret
-  result_fav = @twitter.favorites(count: '200')
+  result_fav = []
+  fav1 = @twitter.favorites(count: '200')
+  fav2 = @twitter.favorites(count: '200', max_id: fav1.last.id)
```
Comment (kamontia) [inline app.rb]: I suppose that it should be specified `fav1.last.id + 1 (or -1)` in `max_id` parameter.
In the current, a list is duplicated at `max_id` boundary.

Comment (chaspy) [reply app.rb]: Exactly!

Comment (kamontia) [reply app.rb]: GOOOOOOD

**Comment by kamontia** [comment]
@chaspy Thank you for your efforts! Nice implementation 😄

**Pull request merged**